MANIFEST.in

```diff
@@ -1,9 +1,12 @@
 include *.py
 include *.txt
+include *.rst
+include .coveragerc
+include tox.ini
```

Comment:
@mgedmin makes a good point. I often find myself looking at tox.ini/.travis.yml to figure out test procedures. I wish I'd been including .travis.yml in the MANIFEST.in all this time (I didn't because I check-manifest didn't complain...I guess it ignores dotfiles, although that's not how its documented).